check_for_new_releases.py

```diff
@@ -50,38 +57,43 @@ def get_repo_url(artifacthub_config) -> str:
                     continue
 
                 try:
-                    release = repo.get_latest_release()
+                    releases_paginated_list = repo.get_releases()
+                    releases = []
```

Comment:
Maybe a list comprehension is a little more readable (at least if you are used to this)

```suggestion
                    min_time  = datetime.datetime.now() - datetime.timedelta(days=args.timeframe)
                    releases =  [release for release in paginated_releases if release.prerelease == False and (args.timeframe == -1 or release.published_at > min_time)]
```
Check if the code works before using it though since I wrote it by heart...